package.json

```diff
@@ -8,34 +8,34 @@
   },
   "scripts": {
     "build:analyze": "npm run build --stats-json && npx --yes webpack-bundle-analyzer dist/storefrontapp/stats.json",
-    "build": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx build storefrontapp --configuration production",
-    "build:core": "nx build core --configuration production",
-    "build:storefrontlib": "nx build storefrontlib --configuration production",
-    "build:asm": "npm --prefix feature-libs/asm run build:schematics && nx build asm --configuration production",
-    "build:user": "npm --prefix feature-libs/user run build:schematics && nx build user --configuration production",
+    "build": "env-cmd --no-override -e dev,b2c,$SPA_ENV npx nx build storefrontapp --configuration production",
```

Comment:
You can revert all the npx usage in package.json. 
Commands in package.json scripts use the local node_modules stuff by default without explicitly using npx.